### PR TITLE
Fix `TIME` using in custom functions of particle shader

### PR DESCRIPTION
**Pull request by Chaosus**
Fix shader crash:
```
shader_type particles;

float time_test(float test) {
	return test + TIME;
}

void process() {
	TRANSFORM[3].x += time_test(0.0001);
}
```

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

File: servers/rendering/renderer_rd/renderer_storage_rd.cpp [hunk 1]
```diff
@@ -10033,7 +10033,7 @@ RendererStorageRD::RendererStorageRD() {
 		actions.renames["RESTART"] = "restart";
 		actions.renames["CUSTOM"] = "PARTICLE.custom";
 		actions.renames["TRANSFORM"] = "PARTICLE.xform";
-		actions.renames["TIME"] = "FRAME.time";
+		actions.renames["TIME"] = "FRAME_TIME";
```
Comment (clayjohn) [inline servers/rendering/renderer_rd/renderer_storage_rd.cpp:10036]:
```suggestion
		actions.renames["TIME"] = "frame_history.data[0].time";
```

I think it is simpler to just use frame_history.data[0].time directly.

Comment (Chaosus) [reply servers/rendering/renderer_rd/renderer_storage_rd.cpp]: Done

**Review by clayjohn — APPROVED** [review]

**Comment by RPicster** [comment]
I found a problem with this PR.
When using an Emitter Shape other than the default (e.g. sphere) the Particles will "die out" with time.
Less and less particles will be spawned over time.

**To reproduce it**
- Create a Particle System
- Set Emitter Shape to sphere
- For quicker visible result increase the `fixed fps` to something high (but also happens with the default 30 fps)

https://user-images.githubusercontent.com/9423774/143681808-4c9872e5-9540-4a02-a441-9910a7691df9.mp4

.

**Comment by Chaosus** [comment]
Please - create a new issue and post a link to this PR if you are sure that it is a regression.

**Comment by RPicster** [comment]
I tried it again on the latest master - and can't seem to reproduce it.

**Pull request merged**